classes/sota.bbclass

```diff
@@ -16,7 +16,8 @@ IMAGE_INSTALL_append_sota = " ostree os-release ${SOTA_CLIENT} ${SOTA_CLIENT_PRO
 IMAGE_CLASSES += " image_types_ostree image_types_ota"
 
 IMAGE_FSTYPES += "${@bb.utils.contains('DISTRO_FEATURES', 'sota', 'ostreepush garagesign garagecheck ota-ext4 wic', ' ', d)}"
-IMAGE_FSTYPES += "${@bb.utils.contains('BUILD_OTA_TARBALL', '1', 'ota-tar ota-tar.xz', ' ', d)}"
+IMAGE_FSTYPES += "${@bb.utils.contains('BUILD_OSTREE_TARBALL', '1', 'ostreetar', ' ', d)}"
```

Comment:
Since BUILD_OSTREE_TARBALL is not set anywhere, there won't be a tar.bz2 anymore by default. This changes behavior... That might be fine, but we should be aware of the fact...

There are some self tests, and one seems to be using tar.bz2... Are the self tests still passing?
See lib/oeqa/selftest/cases/updater.py

Maybe just set BUILD_OSTREE_TARBALL to 1 in the test, or somewhere as a default for now?

Reply:
Yes, I think you are absolutely correct, it will break the test if it does not set BUILD_OSTREE_TARBALL to 1, I will send a V2 to fix this. Thanks for the review.

//Ming Liu